### ISSUE TITLE
[bug] window.btc_providers bug

### ISSUE THREAD

**Issue by orenyomtov** [secretkeylabs/sats-connect#175]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SATS-CONNECT Version

latest

### Current Behavior

There is a bug in this package:
https://www.npmjs.com/package/@sats-connect/make-default-provider-config?activeTab=code

This:
```
    // Rest
    config.options.concat(providers
        .filter((provider) => {
        return (provider.id !== DefaultAdaptersInfo.xverse.id &&
            provider.id !== "unisat");
    })
        .map((provider) => {
        return {
            ...provider,
            ...getInstallPrompt(provider),
        };
    }));
    return config;
```
Should be changed to:
```
    // Rest
    config.options = config.options.concat(providers
        .filter((provider) => {
        return (provider.id !== DefaultAdaptersInfo.xverse.id &&
            provider.id !== "unisat");
    })
        .map((provider) => {
        return {
            ...provider,
            ...getInstallPrompt(provider),
        };
    }));
    return config;
```

### Steps To Reproduce

```
a=[1,2,3]
b=[4,5,6]
console.log(a.concat(b)) // prints 1,2,3,4,5,6
console.log(a) // prints 1,2,3
```

**Comment by aryzing** [comment]
Thanks, fixing it

**Comment by aryzing** [comment]
Should be fixed as of 2.8.1.

**Comment by orenyomtov** [comment]
Wow thanks! So quick

Would it be possible for you to also update the example dapp you're hosting at
https://sats-connect.netlify.app/
?

edit: i built it locally and it works great so no rush, tnx again!
